data/sql/updates/db_world/2019_01_16_00.sql

```diff
@@ -0,0 +1,35 @@
+-- DB update 2019_01_15_00 -> 2019_01_16_00
+DROP PROCEDURE IF EXISTS `updateDb`;
+DELIMITER //
+CREATE PROCEDURE updateDb ()
+proc:BEGIN DECLARE OK VARCHAR(100) DEFAULT 'FALSE';
+SELECT COUNT(*) INTO @COLEXISTS
+FROM information_schema.COLUMNS
+WHERE TABLE_SCHEMA = DATABASE() AND TABLE_NAME = 'version_db_world' AND COLUMN_NAME = '2019_01_15_00';
+IF @COLEXISTS = 0 THEN LEAVE proc; END IF;
+START TRANSACTION;
+ALTER TABLE version_db_world CHANGE COLUMN 2019_01_15_00 2019_01_16_00 bit;
+SELECT sql_rev INTO OK FROM version_db_world WHERE sql_rev = '1547598400341720700'; IF OK <> 'FALSE' THEN LEAVE proc; END IF;
+--
+-- START UPDATING QUERIES
+--
```

Comment:
It will not, since the script was already imported to the updates folder. Check in https://github.com/azerothcore/azerothcore-wotlk/blob/master/apps/db_pendings/import.sh#L34

Reply:
ah yeah